Sign Up Page/signup.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./signup.css" />
+    <title>Sign Up Page</title>
+  </head>
+  <body>
+    <div class="maincon">
+      <h1>Welcome</h1>
+      <h2>Create Account</h2>
+      <form action="./SignUp.html">
```

Comment:
Can you please explain why you write `action="./SignUp.html"` inside the form?

Reply:
Just to write something, here it will go to login page or landing page.